src/ws/shard_manager.ts

```diff
@@ -99,7 +99,15 @@ export function requestAllMembers(
 ) {
   const nonce = `${guild.id}-${Date.now()}`;
   cache.fetchAllMembersProcessingRequests.set(nonce, resolve);
-  return requestGuildMembers(guild.id, guild.shardID, nonce, options);
+
+  const result = await requestGuildMembers(
+    guild.id,
+    guild.shardID,
+    nonce,
+    options,
+  );
+
+  return result;
```

Comment:
```suggestion
  await requestGuildMembers(
    guild.id,
    guild.shardID,
    nonce,
    options,
  );
```
Not returning anything also returns undefined, so basically the assignment is redundant.